modules/auxiliary/scanner/misc/dahua_dvr_auth_bypass.rb

```diff
@@ -0,0 +1,376 @@
+class Metasploit3 < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name'            => %q(Dahua DVR Auth Bypass Scanner),
+      'Description'     => %q(Scans for Dahua-based DVRs and then grabs settings. Optionally resets a user's password and clears the device logs),
+      'Author'          => [
+        'Jake Reynolds - Depth Security', # Vulnerability Discoverer
+        'Tyler Bennett - Talos Infosec' # Metasploit Module
+      ],
+      'References'      => [
+        [ 'CVE', '2013-6117' ],
+        [ 'URL', 'https://depthsecurity.com/blog/dahua-dvr-authentication-bypass-cve-2013-6117' ]
+      ],
+      'License'         => MSF_LICENSE
+       )
+    deregister_options('RHOST')
+    register_options([
+      OptString.new('USERNAME', [true, 'A username to reset', '888888']),
+      OptString.new('PASSWORD', [false, 'A password to reset the user with, if not set a random pass will be generated.']),
+      OptBool.new('VERSION_INFO', ['false', 'Grabs the version of DVR', 'FALSE']),
```

Comment:
For all of these `OptBool` that you added, the first member of the second argument needs to be a boolean or be interpreted as such.  Using `'false'` evaluates to `true`, which is just confusing here.  Change `'false'` to `true`.  Similarly, with the last member of the second argument, `'FALSE'` is as truthy as `'TRUE'`, meaning all of these will default to `true`.  Please change them to `true` or `false` as appropriate.